.changeset/light-beans-speak.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/spinner': major
+'@twilio-paste/core': minor
+---
+
+[Spinner]: Add new spinner component (TODO update this.)
```

Comment:
Maybe we could say something like "Update visual design of the spinner component"? And if we're not introducing a breaking change to the API the consumers use, would this be a major change, or just a minor?

Reply:
Sure thing, this is just a draft for now, hence the "TODO" comment.

Reply:
IMHO since it's a significantly net new component, `major` feels right here, even though the API is the same.

Reply:
I would like to reserve Majors for breaking changes, things that mean the consumer needs to do stuff, just so it's a really clear message on what a major means to a consumer

Reply:
Ok, I'll change to a minor. Thanks folks.